chainer/__init__.py

```diff
@@ -92,6 +136,7 @@ def get_function_hooks():
 global_config.use_cudnn = os.environ.get('CHAINER_USE_CUDNN', 'auto')
 global_config.use_cudnn_tensor_core = 'auto'
 global_config.autotune = False
+global_config.use_ideep = os.environ.get('CHAINER_USE_IDEEP', 'auto')
```

Comment:
Just for remembrance: need to add this to docs.

Reply:
Yes, maybe in another PR.